skycultures/ruanui_sky_tahiti_and_society_islands/index.json

```diff
@@ -0,0 +1,203 @@
+    },
+    {
+      "id": "CON ruanui_sky_tahiti_and_society_islands 013",
+      "lines": [[53740, 54682, 55705, 55282, 53740], [55282, 55687, 56633, 58188, 57283, 55705]],
+      "common_name": {"english": "Vortex ocean in wich to lose crime", "native": "Moana-'ohu-noa-'ei-ha'amoe-hara"}
+    },
+      {
+      "id": "CON ruanui_sky_tahiti_and_society_islands 014",
+      "lines": [[36850, 37826]],
+      "common_name": {"english": "Nā-'ainanu", "native": "Nā-'ainanu"}
+    },
+     {
+      "id": "CON ruanui_sky_tahiti_and_society_islands 015",
+      "lines": [[31681, 34088, 35550, 35350, 32362], [35550, 36962, 37740], [36962, 37826], [36962, 36046, 34693, 36850], [34693, 33018], [34693, 32246, 30883], [32246, 30343, 29655, 28734]],
+      "common_name": {"english": "Open valley", "native": "Fa'a-tā-potupotu"}
+    },
+        {
+      "id": "CON ruanui_sky_tahiti_and_society_islands 016",
+      "lines": [[71683, 68702]],
+      "common_name": {"english": "Two eyes", "native": "Nā-matarua"}
+    },
+      {
+      "id": "CON ruanui_sky_tahiti_and_society_islands 017",
+      "lines": [[26727, 26311, 25930], [29434, 29426], [29434, 28716, 27913], [29426, 29038, 27913], [29426, 28614, 27989, 26727, 27366, 24436, 25930, 25336, 26207, 27989], [25336, 22449, 22549, 22730, 23123], [22449, 22509, 22845], [29038, 28614]],
+      "common_name": {"english": "Forest of parental yearnings", "native": "Uru-meremere"}
+    },
+        {
+      "id": "CON ruanui_sky_tahiti_and_society_islands 018",
+      "lines": [[85927, 86670, 87073, 86228, 84143, 82671, 82514, 82396, 81266, 80763, 80079, 78265, 78401, 78820, 79374, 80343, 80975, 80079]],
+      "common_name": {"english": "Hiro's kite", "native": "Te-'uo-a-hiro"},
+      "visibility": {"months": [5, 8]},
+      "image": {
+"file": "illustrations/kite2.png",
+"size": [365, 456],
+"anchors": [
+{"pos": [318, 32], "hip": 78820},
+{"pos": [239, 135], "hip": 80763},
+{"pos": [132, 391], "hip": 84143}
+]
+}
+    },
+     {
+      "id": "CON ruanui_sky_tahiti_and_society_islands 019",
+      "lines": [[100064, 100345, 104139, 105515, 106985, 107556], [105515, 105881, 104139], [100345, 102485], [104139, 102978]],
+      "common_name": {"english": "Cavern of the parental yearnings", "native": "Rua-o-mere"}
+    },
+    {
+      "id": "CON ruanui_sky_tahiti_and_society_islands 020",
+      "lines": [[42799, 42402, 42313, 43109, 43234, 42799], [43234, 43813, 45336, 46776, 46509, 46390, 48356, 49841, 51069, 52943, 56343, 57936, 64166, 64962]],
+      "common_name": {"english": "Stand in the sea of rank odor", "native": "Tū-i-te-moana-urifā"}
+    },
+       {
+      "id": "CON ruanui_sky_tahiti_and_society_islands 021",
+      "lines": [[85927, 85696]],
+      "common_name": {"english": "The greedy", "native": "Pipiri-ma"}
+    }
+  ],
+ 
+  "common_names": {
+  "HIP 24608": [{"english": "Tahi-ari'i", "references": [1]}],
```

Comment:
These star names should all be tagged as "native". If they have a meaning (something like "red star with two companions") a translation should be given in "english".